Assignment6/src/ProductManager.cpp

```diff
@@ -0,0 +1,61 @@
+#include "ProductManager.h"
+#include "CustomExceptions.h"
+
+bool ProductManager::addProductToInventory(int productId, const std::string& productName, double productPrice, int productQuantity)
```

Comment:
I think here we don't need to return anything in these functions, if something went wrong then there is exception handled by custom exception handling and if won't get any exception or error that means everthing works fine.
I think it should be void return type.

Reply:
I agree with you, but according to the CPP convention it's recommended to return something from a function in order to make the code more test worthy (which includes Gtests). Also, it's generally recommended to use exceptions for exceptional circumstances and booleans for situations where failure is a known or accepted possibility.